src/alto-ui/Dialog/__tests__/Dialog.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Dialog from '../Dialog';
+
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return props => {
```

Comment:
desctucturize props here and you don't need body of function here:
```
jest.mock('../../CloseButton', () => ({ onClick }) => (
  <button className="CloseButton" onClick={() => onClick()} />
);
```

Reply:
Destructurized